cg/models/orders/json_sample.py

```diff
@@ -1,33 +1,32 @@
-from typing import Optional, Any, List
+from typing import Any, List, Optional
 
 from cg.constants import DataDelivery, Pipeline
 from cg.models.orders.sample_base import OrderSample
-from pydantic.v1 import constr, validator
+from pydantic import BeforeValidator, constr
+from typing_extensions import Annotated
+
+
+def join_list(value: Any):
+    return "".join(value) if isinstance(value, list) else value
+
+
+def convert_well(value: str):
```

Comment:
Can we add a docstring what it is we are converting more specifcally? What should the layout be of a "converted_well" ?